src/blueteam/TeamConstants.java

```diff
@@ -23,5 +26,11 @@
 	 * are reserved for this.
 	 */
 	public static final int ROBOT_COUNTERS_BEGIN = 0;
-	public static final int MINIMUM_BULLETS_TO_SAVE = 300;
+	public static final int MINIMUM_BULLETS_TO_SAVE = 150;
+
+	public static final int GARDENERS_DIRECT_PATH_LENGTH = 30;
+	public static final float GARDENERS_DEFAULT_FREE_SPOT_RADIUS = 6.5f;
+
+	public static final Direction GARDENERS_GARDEN_ENTRANCE = Direction.WEST;
+	public static final int GARDENER_NUM_OF_TREES_TO_BUILD_LUMBER = 5;
```

Comment:
Mby tweak this too. @Lukx19 GARDENER_NUM_OF_TREES_TO_BUILD_LUMBER should be smaller, to avoid no lumberjack is builded, or gardener lumberjack building should be reworked....